mne_connectivity/tests/test_connectivity.py

```diff
@@ -118,11 +121,22 @@ def test_connectivity_containers(conn_cls):
     conn.rename_nodes(lambda x: '0' if x == 'new_name' else x)
     assert_array_equal(orig_names, conn.names)
 
-    conn2 = conn_cls(data=correct_numpy_input, n_nodes=3, indices=indices,
-                     **extra_kwargs)
-    conn3 = conn_cls(data=correct_numpy_input, n_nodes=3, indices=indices,
-                     **extra_kwargs)
-    np.testing.assert_array_equal(conn2.get_data(), conn3.get_data())
+    indiced_numpy_shape, indice_kwargs = _prep_correct_connectivity_input(
+        conn_cls, n_nodes=n_nodes, symmetric=False, n_epochs=n_epochs,
+        indices=indices
+    )
+    indiced_numpy_input = np.ones(indiced_numpy_shape)
+    conn2 = conn_cls(data=indiced_numpy_input, n_nodes=3, indices=indices,
+                     **indice_kwargs)
+    conn3 = conn_cls(data=indiced_numpy_input, n_nodes=3, indices=indices,
+                     **indice_kwargs)
+    assert_array_equal(
+        conn2.get_data(output='dense'), conn3.get_data(output='dense'))
```

Comment:
Not sure what you are testing here, is `conn2` and `conn3` not exactly the same? If this is correct, add a comment on what's happening to make things clear for future reference.

Reply:
Oh you're right. There was somehow a mistake here where n_nodes is set to 3 in both cases. This tests when n_nodes is different and indices and data are the same. I added a comment to clarify that